src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/EncoderExtensions.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.IO;
 
-namespace System.Text.Encodings.Web
+namespace Microsoft.Framework.WebEncoders
```

Comment:
There are many APIs in progress in this component and lots of them will be changed or removed. I don't think there is need to track this with TODOs, but I added issue #1585 to make sure the removals are tracked.